railties/lib/rails/test_unit/runner.rb

```diff
@@ -44,12 +45,20 @@ def run(argv = [])
         def load_tests(argv)
           patterns = extract_filters(argv)
 
-          tests = Rake::FileList[patterns.any? ? patterns : "test/**/*_test.rb"]
-          tests.exclude("test/system/**/*", "test/dummy/**/*") if patterns.empty?
+          tests = Rake::FileList[patterns.presence || default_test_glob]
```

Comment:
Do we really need to use `presence` here? The previous code were just fine.

Reply:
@jeremy added it in his initial suggestion, I removed it since no active support extension, and then @kaspth suggested adding active support extensions so I did 😂. I'll remove it